src/lib/gssapi/krb5/init_sec_context.c

```diff
@@ -118,13 +118,14 @@ int krb5_gss_dbg_client_expcreds = 0;
  * ccache.
  */
 static krb5_error_code get_credentials(context, cred, server, now,
-                                       endtime, out_creds)
```

Comment:
IMO, the sooner we start fixing the K&R prototypes to C89+ prototypes, the better...

Maybe instead of `probe` you could pass in `flags`.